### ISSUE TITLE
Add 'moved' event for camera

### ISSUE THREAD

**Issue by JohnSundell** [JohnSundell/ImagineEngine#86]
Imagine Engine offers an event system to enable users to trigger logic based on when something happened in a scene. For example, here's how an `Actor` can be observed, triggering a closure whenever it's moved:

```swift
actor.events.moved.observe {
    // The actor was moved
}
```

We should add a similar event to `Camera` as well, so that you can do this:

```swift
camera.events.moved.observe {
    // The camera was moved
}
```

**Things to do:**

- `Camera` does not yet support events, so a `CameraEventCollection` will need to be created.
- Add a `moved` event (similar to how it works for `Actor`).
- Trigger the moved event whenever a camera's position was changed.

**Good to know:**

- Each `Scene` has a `Camera` (see the `camera` property).
- See `ActorEventCollection` to see how events are defined for `Actor`.
- For inspiration on how to unit test this change, see `ActorTests`, which include several tests that involve events.

**Comment by kkapitan** [comment]
Hi! I've submitted PR for this issue #97 :)

**Comment by JohnSundell** [comment]
Fixed by @kkapitan in #97 🎉
